package.json

```diff
@@ -18,7 +18,7 @@
     "test:a11y": "echo 'TODO: Accessibility tests'",
     "test:nsp": "nsp check",
     "test:cucumber": "protractor test/config/cucumber.conf.js",
-    "test:smoke": "protractor test/config/smoke.conf.js",
+    "test:smoke": "echo 'TODO: Enable Smoke tests'",
```

Comment:
have we disabled smoke tests?

Reply:
This could be a matter of this branch being out-of-sync with the `master` branch.
@MKaramath could you please merge `master` in this branch and double-check that this pull request doesn't contains any changes on `package.json`